tests/e2e/test_stats.py

```diff
@@ -0,0 +1,72 @@
+from .base_testcase import BaseTestCase
+from selenium.webdriver.common.by import By
+import os
+import unittest
+
+class TestStats(BaseTestCase):
+    def __init__(self, testname):
+        super().__init__(testname, log_in=False)
+    def individual_grading_stats_test_helper(self, user_id, full_access):
+        self.log_in(user_id=user_id)
+        self.click_class('sample')
+        self.driver.find_element_by_xpath('//a[contains(@href,"/sample/gradeable/grading_homework/grading/status")]').click()
+        numerical_data_text = self.driver.find_element_by_id("numerical-data").text
+        if full_access:
+            self.assertTrue("Students who have submitted: 72 / 101 (71.3%)" in numerical_data_text)
+            self.assertTrue("Current percentage of grading done: 43.75 / 72 (60.8%)" in numerical_data_text)
+            self.assertTrue("Section 1: 4 / 12 (33.3%)" in numerical_data_text)
+        else:
+            self.assertTrue("Students who have submitted: 15 / 20 (75%)" in numerical_data_text)
+            self.assertTrue("Current percentage of grading done: 9 / 15 (60.0%)" in numerical_data_text)
+            self.assertTrue("Section 4: 4 / 8 (50.0%)" in numerical_data_text)
+        self.log_out()
+    def individual_released_stats_test_helper(self, user_id, full_access):
+        self.log_in(user_id=user_id)
+        self.click_class('sample')
+        self.driver.find_element_by_xpath('//a[contains(@href,"/sample/gradeable/grades_released_homework/grading/status")]').click()
+        numerical_data_text = self.driver.find_element_by_id("numerical-data").text
+        if full_access:
+            self.assertTrue("Students who have submitted: 71 / 101 (70.3%)" in numerical_data_text)
+            self.assertTrue("Current percentage of grading done: 71 / 71 (100.0%)" in numerical_data_text)
+            self.assertTrue("Section 1: 8 / 8 (100.0%)" in numerical_data_text)
+            self.assertTrue("Number of students who have viewed their grade: 53 / 71 (74.6%)" in numerical_data_text)
+        else:
```

Comment:
I'm failing this test here.  When I look manually at `grades_released_homework` as instructor I see:

![image](https://user-images.githubusercontent.com/7605020/90642118-fd409680-e1ff-11ea-8522-e4b672112fa3.png)

Is it possible your assertion is greater than it should be?